ietf/templates/meeting/session_details_panel.html

```diff
@@ -66,22 +66,20 @@ <h3 class="mt-4">Agenda, Minutes, and Bluesheets</h3>
                             <td>
                                 <a href="{{ pres.document.get_href }}">{{ pres.document.title }}</a>
                                 <a href="{{ url }}">({{ pres.document.name }})</a>
-                            </td>
-                            <td class="text-end">
                                 {% if user|has_role:"Secretariat" or can_manage_materials %}
+                                    {% if pres.document.type.slug != 'bluesheets' or user|has_role:"Secretariat" or meeting.type.slug == 'interim' and can_manage_materials %}
```

Comment:
Why did this block move?

Reply:
So the float-end looks better.

Reply:
I don't see, at this point, how the float-end could have been affected?